firerunner/src/main.rs

```diff
@@ -109,8 +144,16 @@ fn main() {
     let shared_info = Arc::new(RwLock::new(InstanceInfo {
         state: InstanceState::Uninitialized,
         id: instance_id,
+        load_dir: None,
```

Comment:
Instead of starting with `None` and writing later based on the if-branch, just:

```rust
    load_dir: cmd_arguments.value_of("dump_dir").map(PathBuf::from),
    dump_dir: cmd_arguments.value_of("load_dir").map(PathBuf::from),
```